spring-cloud-kubernetes-commons/pom.xml

```diff
@@ -50,13 +50,29 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
+			<exclusions>
+				<exclusion>
```

Comment:
drop it and include `mockito-inline`, for the reasons below. (it includes `mockito-core` internally anyway, so this is a non-breaking change)